db/version_set.cc

```diff
@@ -1640,8 +1640,16 @@ void VersionStorageInfo::ComputeCompactionScore(
           level_bytes_no_compacting += f->compensated_file_size;
         }
       }
-      score = static_cast<double>(level_bytes_no_compacting) /
-              MaxBytesForLevel(level);
+      uint64_t max_bytes = MaxBytesForLevel(level);
+      if (max_bytes == 0) {
+        score = static_cast<double>(level_bytes_no_compacting) /
+                mutable_cf_options.max_bytes_for_level_base;
+        // This level is no longer necessary so make sure it is always eligible
+        // for compaction.
+        score = std::max(score, 1.0);
```

Comment:
If we want to prioritize cleaning up higher levels, we could vary the minimum score. Example:
```cpp
score = std::max(score, 1.0 + static_cast<double>(MaxInputLevel() - level) / 10);
```

Reply:
I agree that 1.0 maybe a little bit small. We don't want to fir compact L4->L5 while we are still accumulating data L0->L3, and we are going to compact L3->L4 anyway. I would say something like 1.5 or so feels OK to me, if we don't want to make things too complicated.